portfolio/src/main/webapp/script.js

```diff
@@ -49,9 +49,23 @@ function randomFood() {
 
 /** 
  * Retrives the text on the .../data URL and adds it to the DOM
+ * Edit so that it shows in stacked form like on /data page
  */
 function getData() {
-  fetch('/data').then(response => response.text()).then((quote) => {
-    document.getElementById('data-container').innerHTML = quote;
+  fetch('/data').then(response => response.json()).then((users) => {
+    const printCom = document.getElementById('data-container');
+    users.forEach((user) => {
+    printCom.innerHTML = '';
```

Comment:
resetting printCom.innerHTML here clears comments which were added in earlier iterations of this loop. So this only displays the last comment in 'users'. Is that intended?

Reply:
I fixed this in a later CL but you are correct it wasn't